src/backend/opencl/blas_clblas.cpp

```diff
@@ -7,6 +7,8 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+#if defined(USE_CLBLAS)
```

Comment:
I agree, that would be my preference as well. But it seems ArrayFire is using the *.cpp GLOB in CMake: `FILE(GLOB opencl_sources "*.cpp")`, so I'm not sure how to handle that properly without explicitly listing all files...